testnet/stacks-node/src/nakamoto_node/sign_coordinator.rs

```diff
@@ -657,7 +706,88 @@ impl SignCoordinator {
                 continue;
             }
 
-            // TODO: get messages from signers (#4775)
+            let modified_slots = &event.modified_slots.clone();
+
+            // Update `next_signers_bitvec` with the slots that were modified in the event
```

Comment:
I don't think we want to do this here.  If we're mutating the bitvec _after_ we get a signer signature, then it would mean that the signer would need to re-sign the block with the new bitvec.

Instead, the miner needs to obtain the signer-determined bitvec _a priori_, and just put it directly into the block.  How exactly this happens is still undetermined (it'll be a separate PR that can ship after Nakamoto), but the miner shouldn't be in the business of modifying the signer bitvec due to the above.

Reply:
Yeah, I had added this in the `begin_sign_v1` code a while back, and the purpose was to set up the ability for the miner to potentially use it in the _next_ Nakamoto block. If I'm thinking the same way, I think that would be the "a priori" approach, but only utilized in the next block. This `next_signers_bitvec` is never actually used, though, so I'm fine with taking it out.

There are lots of gotchas here - like the signer's signature could come in right after the miner reaches the threshold, and so it doesn't really make sense to remove them from the Bitvec in the next block, because that signer is just going to reject the block. This is actually a good potential use case for `next_signer_bitvec` - the miner got a response from the signer, but the block was rejected, so the miner retries the same block with an update BitVec.

Overall fine with punting on this, as this is just an unused feature at the moment.

Reply:
Context from the PR that added this: https://github.com/stacks-network/stacks-core/pull/4627

Reply:
Removed 👍🏼